File: Python Fundamentals/4. Functions - Exercise/04_odd_and_even_sum.py
def sums(number):
    even_sum = 0
    odd_sum = 0
    for i in number:
        new_i = int(i)
        if new_i % 2 == 0:
            even_sum += new_i
        else:
            odd_sum += new_i
    print(f"Odd sum = {odd_sum}, Even sum = {even_sum}")


current_number = input()
sums(current_number)

File: 5. JS Front-End/JS Front-End - Exercises/3. Functions and Statements - Exercises/04_odd_and_even_sum.js
function evenAndOddsums(number) {
    strNumber = number.toString()

    oddSum = 0
    evenSum = 0

    for (let digit of strNumber) {
        let numDigit = Number(digit)

        if (numDigit % 2 !== 0) {
            oddSum += numDigit
        } else {
            evenSum += numDigit
        }
    }

    return `Odd sum = ${oddSum}, Even sum = ${evenSum}`
}







// Test code
// console.log(evenAndOddsums(1000435))